chips/earlgrey/src/plic.rs

```diff
@@ -2,6 +2,7 @@
 
 use kernel::common::registers::{register_bitfields, register_structs, ReadOnly, ReadWrite};
 use kernel::common::StaticRef;
+use kernel::debug;
```

Comment:
Why is this needed?

Reply:
From when I was discovering that USBDEV was issuing connected interrupts. I'll clean this (and all of the other debug code) up before being ready for merging.

Reply:
I didn't realise this wasn't intended to be ready for merging. Why not mark it as a draft PR then?

Reply:
Ah, you do mention it in the description